srml/democracy/src/lib.rs

```diff
@@ -318,6 +391,11 @@ impl<T: Trait> Module<T> {
 	}
 
 	/// Get the voters for the current proposal.
+	///
+	/// # <weight>
+	/// - **FLAG**
+	/// - Dependent on the size of `voters_for`, could the array be inflated?
```

Comment:
Voting requires locking funds, so although it can grow quite large, there are funds at stake.
```suggestion
	/// - Dependent on the size of `VotersFor`, could the vector be inflated?
```